packages/driver/src/cy/mouse.ts

```diff
@@ -567,6 +567,21 @@ export const create = (state: StateFunc, keyboard: Keyboard, focused: IFocused,
           return { skipClickEventReason: 'element was detached' }
         }
 
+        // Only send click event if element is not disabled.
+        // First find an parent element that can actually be disabled
+        const canBeDisabled = (el) => {
+          const actuallyDisabledElements = ['button', 'input', 'select', 'textarea', 'optgroup', 'option', 'fieldset']
+
+          return actuallyDisabledElements.indexOf($elements.getTagName(el)) > -1
+        }
```

Comment:
I had some issues getting this code to run. In the current state, `canBeDisabled` returns a `boolean` when `findParent` is expecting an element. the boolean is passed into `$elements.isDisabled` and is failing since it's expecting a JQuery selector. Were you able to get this to run OK? Either way I think we can get it working with the following suggestions!

```suggestion
        const findParentThatCanBeDisabled = (el: HTMLElement): HTMLElement | null => {
          const elementsThatCanBeDisabled = ['button', 'input', 'select', 'textarea', 'optgroup', 'option', 'fieldset']

          return elementsThatCanBeDisabled.includes($elements.getTagName(el)) ? el : null
        }
        }
```